framework/mesh/mesh_continuum/mesh_continuum.h

```diff
@@ -38,27 +38,11 @@ class MeshContinuum
   LocalCellHandler local_cells;
   GlobalCellHandler cells;
 
-private:
-  MeshAttributes attributes = NONE;
-
-  struct
-  {
-    size_t Nx = 0;
-    size_t Ny = 0;
-    size_t Nz = 0;
-  } ortho_attributes;
-
-  std::map<uint64_t, std::string> boundary_id_map_;
-
 public:
-  MeshContinuum()
-    : local_cells(local_cells_),
-      cells(local_cells_,
-            ghost_cells_,
-            global_cell_id_to_local_id_map_,
-            global_cell_id_to_nonlocal_id_map_)
-  {
-  }
+  MeshContinuum();
+
+  unsigned Dimension() const { return dim_; }
```

Comment:
```suggestion
  unsigned int Dimension() const { return dim_; }
```
Not that it makes a difference, but I don't believe we've used `unsigned` anywhere else in the code. This could maybe be added to the coding standards (either with or without `int`). I'm indifferent as to which to use.